### ISSUE TITLE
java.nio.file.NoSuchFileException – marvel log files

### ISSUE THREAD

**Issue by grebett** [grebett/learning-elasticsearch#1]
Although Elasticsearch launches correctly, using it with marvel leads to a `java.nio.file.NoSuchFileException` error. It seems that some `.tlog` files are missing. Error pasted here:

``` bash
java.nio.file.NoSuchFileException: /Users/grebett/Work/42/Matcha/Playground/elasticsearch-2.1.0/data/elasticsearch/nodes/0/indices/.marvel-es-data/0/translog/translog-4962244458463018733.tlog
    at sun.nio.fs.UnixException.translateToIOException(UnixException.java:86)
    at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:102)
    at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:107)
    at sun.nio.fs.UnixFileSystemProvider.implDelete(UnixFileSystemProvider.java:244)
    at sun.nio.fs.AbstractFileSystemProvider.delete(AbstractFileSystemProvider.java:103)
    at java.nio.file.Files.delete(Files.java:1126)
    at org.elasticsearch.index.translog.Translog.recoverFromFiles(Translog.java:324)
    at org.elasticsearch.index.translog.Translog.<init>(Translog.java:166)
    at org.elasticsearch.index.engine.InternalEngine.openTranslog(InternalEngine.java:209)
    at org.elasticsearch.index.engine.InternalEngine.<init>(InternalEngine.java:152)
    at org.elasticsearch.index.engine.InternalEngineFactory.newReadWriteEngine(InternalEngineFactory.java:25)
    at org.elasticsearch.index.shard.IndexShard.newEngine(IndexShard.java:1408)
    at org.elasticsearch.index.shard.IndexShard.createNewEngine(IndexShard.java:1403)
    at org.elasticsearch.index.shard.IndexShard.internalPerformTranslogRecovery(IndexShard.java:906)
    at org.elasticsearch.index.shard.IndexShard.performTranslogRecovery(IndexShard.java:883)
    at org.elasticsearch.index.shard.StoreRecoveryService.recoverFromStore(StoreRecoveryService.java:245)
    at org.elasticsearch.index.shard.StoreRecoveryService.access$100(StoreRecoveryService.java:56)
    at org.elasticsearch.index.shard.StoreRecoveryService$1.run(StoreRecoveryService.java:129)
    at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
    at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
    at java.lang.Thread.run(Thread.java:745)
```

**Comment by grebett** [comment]
According to [stackoverflow](http://stackoverflow.com/questions/33733253/elasticsearch-error-no-such-file-exception), marvel may be included in Elasticsearch 2.0, which is quite weird since [official documentation](https://www.elastic.co/guide/en/marvel/current/installing-marvel.html) ask us to install it as a plugin.

This error does not seem to prevent Marvel from working. Error closed till more informations...

**Comment by grebett** [comment]
After more search, this issue is known and explained here:
https://github.com/elastic/elasticsearch/pull/14872
